app/assets/javascripts/components/stories/Sprints.js

```diff
@@ -1,32 +1,48 @@
-import React from "react";
-import PropTypes from "prop-types";
-import Sprint from "./Sprint";
+import React from 'react';
+import PropTypes from 'prop-types';
+import Sprint from './Sprint';
+import { Droppable } from 'react-beautiful-dnd';
 
 const propTypes = {
-  sprints: PropTypes.array
+  sprints: PropTypes.array,
 };
 
 const defaultProps = {
   sprints: [],
 };
 
-const renderSprints = (sprints, fetchStories) => {
-  return sprints.map(
-    (sprint, index) =>
-      sprint ? (
-        <Sprint
-          key={sprint.number}
-          sprint={sprint}
-          fetchStories={fetchStories}
-        />
-      ) : null
-  );
-};
+const droppableContainer = (sprints, fetchStories, columnId) => (
+  <Droppable droppableId={JSON.stringify({columnId, sprintIndex: 0})} isDropDisabled={columnId === 'done'}>
+    {provided => (
+      <div className='Sprints' ref={provided.innerRef} {...provided.droppableProps}>
+        {renderSprints(sprints, fetchStories, columnId)}
+        {provided.placeholder}
+      </div>
+    )}
+  </Droppable>
+);
```

Comment:
This looks a bit a odd, if you are calling this when `sprints.length === 0` why are you passing the `sprints` and calling `renderSprints` ?

Reply:
I din't notice that. thanks.